### DISCUSSION TITLE
FreeBSD packages.

### DISCUSSION THREAD

**Discussion by z4kz** [ravynsoft/ravynos#208] [Q&A] [answered]
Will FreeBSD packages be supported on airyx?

**Comment by mszoek** [comment, marked as answer]
They will! I'll reinstate the FreeBSD package repo as soon as I solve the issue of them sometimes breaking the base OS with different library versions and such.

**Reply by z4kz** [reply]
nice, thanks!

**Reply by Retia-Adolf** [reply]
Wanna try ravynOS as a daily driver, but wondering how its compatibility goes now?

I see the release page says: 

> Very few apps available

And since it also says ravynOS has moved to WindowServer, if I understand it correctly, does it mean x11 support is dropped? Would it make GUI programs from Linux binary or FreeBSD ports largely unusable at the moment?

**Reply by mszoek** [reply]
It's not ready to use as a daily driver yet. WindowServer is based on direct rendering and wayland, so X apps will not run on it yet. (The plan is to implement Xwayland once the native desktop is more complete.) Only wayland apps that do client-side decorations or native Cocoa apps built with AppKit can run on it, and there aren't many. You're certainly welcome to help get more stuff implemented though!
